pkg/storage/snapshot/restore_test.go

```diff
@@ -438,11 +439,13 @@ var _ = Describe("Restore controller", func() {
 				r := createRestoreWithOwner()
 				refs := r.OwnerReferences
 				r.OwnerReferences = nil
+				r.Status = nil
```

Comment:
Hi I noticed that I missed doing r.Finalizers = nil here and add a check that the updated restore was added the finalizer, do you mind adding it?

Reply:
Turns out this test was not working as expected so limited scope

Reply:
limiting the scope is good, but now if Im not missing something we miss a test that makes sure we add the finalizer and owner

Reply:
Okay added this: https://github.com/kubevirt/kubevirt/blob/fce03148c1b1f0b266abbbc531e42ce8f31d01b1/pkg/storage/snapshot/restore_test.go#L461-L499

It appears to be already tested for the "restore to a new VM" case